src/popup/router/components/Modals/AssetSelector.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <Modal
+    full-screen
+    has-close-button
+    class="asset-selector"
+    @close="handleClose"
+  >
+    <template #header>
+      {{ $t('pages.fungible-tokens.select-asset') }}
+    </template>
+    <SearchBar
```

Comment:
I checked this component and it uses a weird hybrid of the old UI with new UI. So in practice when user focuses the field it gets bigger and the content below "jumps". Maybe it would be good to use the `new-ui` prop here and pass it to child or just make the SearchBar natively new-ui.